app/views/articles/shared/_article_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_with model: @article, class: 'article-form' do |form| %>
```

Comment:
It is better not to use instance variables in partials, use local variables instead.